README.md

```diff
@@ -90,10 +95,11 @@ A Sawppy builder can now choose from many Sawppy software options. Roughly in or
 * **Original**: HTML-based wireless teleoperation software stack [modified from SGVHAK rover](/docs/SGVHAK%20Rover%20Software.md). This was written to be easy for others to understand and modify.
 * **R/C control**: [lightly modified from my SGVHAK rover software](https://github.com/mw46d/SGVHAK_Rover) by Marco Walther (mw46d) for a Raspberry Pi-based way to interface with traditional remote control receivers.
 
-Plus two options for turning Sawppy into a [ROS](http://ros.org) robotics platform.
+Plus three options for turning Sawppy into a [ROS](http://ros.org) robotics platform.
```

Comment:
Instead of "three", replace with "several" so we don't have to keep updating the numbers as we add to the list.

Reply:
Sorry I'm unsure how to proceed here. I think this commit https://github.com/Roger-random/Sawppy_Rover/pull/28/commits/d8508ea9f489d97a6f553b64cab2f311f54e6691 is the only one relevant to your upstream repo. 
As I've since made changes to my repo that are not relevant upstream I don't see a way to fix it. 
If I make the three > several change in my repo, then do another pull then you'll get all the irrelevant Ag stuff again. In retrospect what I think I should have done is made my 'Ag' changes in a /dev and then merged that back into my /main after this pull was accepted?

Reply:
There are lots of different strategies for managing changes, feel free to adopt one that suits you. What you describe with a /dev branch could work.